pyproject.toml

```diff
@@ -24,7 +24,7 @@ url = "https://pypi.org/simple/"
 secondary = true
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.7"
```

Comment:
If we support 3.7, we should test it in the CI

Reply:
Agreed! I looked. It was there at the time. Although now that I think further through it, there were 11 commits pulled down.